libevmasm/Assembly.h

```diff
@@ -114,6 +116,8 @@ class Assembly
 
 	struct OptimiserSettings
 	{
+		static size_t constexpr DefaultExecutions = frontend::OptimiserSettings::DefaultExecutions;
```

Comment:
This creates a dependency between `libevmasm` and `libsolidity` and @axic wanted to avoid that in #10286 so we'll probably want a hard-coded 200 here.

Or maybe I should just create some global file for constants?

Reply:
But there is a translation anyway between the two settings, right? So maybe the settings are more low-level than both. We could also take the default from here if that is better.

Reply:
I'm not sure there is a translation already. I thought that was the point of #10286. In that PR I added a constructor that can initialize one `OptimiserSettings` type from the other and @axic wanted the previous version of the code to be restored - I thought it was because of this dependency?

I think we should get an opinion from @axic here.